css/main.scss

```
@@ -1096,7 +1096,17 @@ body.documentation{
 		width: calc(100% - 252px);
 		display: flex;
 		justify-content: space-between;
-		align-items: center;
+		align-items: right;
```

```suggestion
		align-items: center;
```
// this is the vertical alignment, likely reverting this makes `margin-top:1px` redundant.